lib/bap/bap.mli

```diff
@@ -8502,6 +8502,12 @@ module Std : sig
     *)
     val compute_liveness : t -> (tid, Var.Set.t) Solution.t
 
+
+    (** [flatten sub] returns [sub] in flattened form in which
+        Let expressions and compound expressions are flattened
+        into sequences of simple definitions. *)
```

Comment:
```suggestion
    (** [flatten sub] returns [sub] in flattened form in which all operands are trivial.
        @see Blk.flatten for more information about flattening. 
        
        @since 2.5.0 *)
```